src/geo/lng_lat.js

```diff
@@ -73,6 +73,24 @@ class LngLat {
     }
 
     /**
+     * Returns a `LngLatBounds` from the coordinates extended by a given `radius`.
+     *
+     * @param {number} radius Distance in meters from the coordinates to extend the bounds.
+     * @returns {LngLatBounds} A new `LngLatBounds` object representing the coordinates extended by the `radius`.
+     * @example
+     * var ll = new mapboxgl.LngLat(-73.9749, 40.7736);
+     * ll.toBounds(100).toArray(); // = [[-73.97501862141328, 40.77351016847229], [-73.97478137858673, 40.77368983152771]]
+     */
+    toBounds(radius: number) {
+        const latAccuracy = 360 * radius / 40075017,
+            lngAccuracy = latAccuracy / Math.cos((Math.PI / 180) * this.lat);
+
+        const LngLatBounds = require('./lng_lat_bounds');
```

Comment:
Yeah explained at https://github.com/mapbox/mapbox-gl-js/pull/3737#issuecomment-266634879 it was due to circular dependency issues which arose when it was outside the function :wink: